sharktank/sharktank/utils/export_artifacts.py

```diff
@@ -529,4 +530,4 @@ def export_and_compile_llm(
         """
         self.export_llm_to_mlir(skip_decode=skip_decode)
         self.compile_to_vmfb(extra_args=extra_compile_args, hal_dump_path=hal_dump_path)
-        return str(self.output_vmfb.resolve())
+        return str(Path(self.output_vmfb).resolve())
```

Comment:
All of these paths should have been Paths rather than strings. I should have forced them to be paths in the initializer. Then this and the other defensive conversions to Path are not needed